src/Nox.Ui.Blazor.Lib/Models/GenericEntityModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Nox.Types;
+namespace Nox.Ui.Blazor.Lib.Models;
+
+public class GenericEntityModel
```

Comment:
I would rename it to SelectEntityModel, as it's used in the select list. GenericEntityModel is too generic for me :)